content/frameworks/react/guides.yaml

```diff
@@ -0,0 +1,32 @@
+title: React
+description: Build performant and progressive React applications
+overview: |
+    In this section you'll learn how to use built-in APIs and third-party 
+    dependencies to improve the performance of your React applications.
+ order: 7
+
```

Comment:
I notice that this table of contents doesn't totally match the list of guides at the end of Get Started. Is that intentional? It's a little confusing. I'm gonna add another comment to the get started guide, as maybe you will want to cover topics beyond performance?

Reply:
Yeah I wasn't sure if I should include all of the guides in the table of content, but I think you're right.

I've removed the one, non-performant-specific guide, from the yaml file and tried to match things to exactly here. 

I've also added links in case readers want to jump directly into one from here, but let me know if that's not a good idea.